templates/institutional/institution-landing-en.html

```diff
@@ -0,0 +1,182 @@
+---
+title: "[Institution-landing-page]"
+layout: no-container
+language: "en"
+altLangPage: "institution-landing-fr.html"
+breadcrumb: [
+	{ "title": "Canada.ca", "link": "https://www.canada.ca/en.html" },
+	{ "title": "Departments & agencies", "link": "https://www.canada.ca/en/government/dept.html" }
+]
+dateModified: "2021-06-01"
+share: true
+followUsLayout: "inline"
+---
+<div class="bg-center bg-cover" data-bgimg-srcset="img/1x1.png 991w, img/ip-cover-image-1200x726.jpg 992w">
+	<div class="container">
+		<div class="row mrgn-bttm-md">
+			<div class="col-md-7">
+				<h1 property="name" id="wb-cont">[Institution name]</h1>
+				<p>Short description of the institution’s mandate.</p>
+				<a class="btn btn-call-to-action" href="#">Super task button [optional]</a>
+			</div>
+		</div>
+	</div>
+</div>
+
+<div class="provisional gc-featured-link">
+	<div class="container">
+		<div class="row">
+			<div class="col-xs-12">
+				<h2 class="wb-inv">Featured link</h2>
+				<p><a class="stretched-link" href="#">Featured link [optional]</a></p>
+			</div>
+		</div>
+	</div>
+</div>
+
+<section class="well well-sm provisional gc-most-requested">
+	<div class="container">
+		<div class="row">
+			<div class="col-md-2">
+				<h2>Most requested</h2>
+			</div>
+			<div class="col-md-10">
+				<ul class="colcount-md-2">
+					<li><a href="#">[Top task hyperlink]</a></li>
+					<li><a href="#">[Top task hyperlink]</a></li>
+					<li><a href="#">[Top task hyperlink]</a></li>
+					<li><a href="#">[Top task hyperlink]</a></li>
+					<li><a href="#">[Top task hyperlink]</a></li>
+					<li><a href="#">[Top task hyperlink]</a></li>
+				</ul>
+			</div>
+		</div>
+	</div>
+</section>
```

Comment:
Not a blocker, but what about to use a Jekyll include provided by that component?

Reply:
@duboisp totally agree and I suggest doing this in a near future and to familiarize myself with these type of includes, I volunteer for this task.